nexus/db-queries/src/db/datastore/trust_quorum.rs

```diff
@@ -65,39 +65,55 @@ fn epoch_to_i64(epoch: Epoch) -> Result<i64, Error> {
 }
 
 impl DataStore {
-    /// Return all `HwBaseboardId`s for a given rack that has run LRTQ
+    /// Return a `RackUuid` for each trust quorum along with its latest `Epoch`.
     ///
-    /// No need for pagination, as there at most 32 member sleds per rack
-    pub async fn lrtq_members(
+    /// For now, since we do not have multirack, and we aren't sure how big
+    /// those clusters are going to be we return all values and don't bother
+    /// paginating. The current `SQL_BATCH_SIZE` is also 1000, and it's unlikely
+    /// that there will ever be more than 1000 racks in a single fleet, sharing
+    /// a single CRDB cluster.
+    pub async fn tq_get_all_rack_id_and_latest_epoch(
         &self,
         opctx: &OpContext,
-        rack_id: RackUuid,
-    ) -> ListResultVec<HwBaseboardId> {
+    ) -> ListResultVec<(RackUuid, Epoch)> {
```

Comment:
Would returning a `BTreeMap<RackUuid, Epoch>` be more clear? The returned vec can't have any duplicate `RackUuid`s, but the caller can't tell that from the type.

Reply:
Yes, that's much better. I'll do that.